staff/jordi-pujol/coworking/coworking-app/src/logic/index.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+import coworkingApi from '../coworking-api'
+
+/**
+ * Abstraction of business logic.
+ */
+const logic = {
+    __coworkingApiToken__: null,
+    __isAdmin__: false,
+
+    /**
+    * Registers a user.
+    * 
+    * @param {string} name 
+    * @param {string} surname 
+    * @param {string} email 
+    * @param {string} password 
+    * @param {string} passwordConfirmation 
+    */
+    registerUser(name, surname, email, password, passwordConfirm) {
+        if (typeof name !== 'string') throw TypeError(name + ' is not a string')
```

Comment:
SEE `staff/manuel-barzi/music-validations`. may you use a module like that to apply validations in a generic way as in `staff/manuel-barzi/staff/music-api/src/logic/index.js`. check it out!